src/components/Transcript/PhraseElm.ts

```diff
@@ -42,8 +42,13 @@ export class PhraseElement implements RedomComponent {
 
   private handleClick = (e: MouseEvent) => {
     e.stopPropagation();
+    const containerCoordinates = document.querySelector('.transcript-section').getBoundingClientRect();
```

Comment:
Encapsulation violation. You cannot directly inspect the bowels of your parent.

Reply:
Done using parent element as discussed.